Update of all Jitsi Meet components

ajs124 opened the pull request:
###### Description of changes
This updates all of the Jitsi Meet components in nixpkgs to their latest stable versions. 

The test does not succeed either. Hence draft.

Closes #200739, #200738, #200736, #200735 and #198793

###### Things done
- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

r-ryantm committed:
jitsi-meet: 1.0.6644 -> 1.0.6943

r-ryantm committed:
jitsi-meet-prosody: 1.0.6644 -> 1.0.6943

r-ryantm committed:
jicofo: 1.0-940 -> 1.0-987

r-ryantm committed:
jitsi-videobridge: 2.2-45-ge8b20f06 -> 2.2-69-gad606ca2

ajs124 committed:
jibri: 8.0-139-g7ab9aa2 -> 8.0-140-gccc7278

wegank commented:
@ofborg build jibri jibri.passthru.tests jicofo jicofo.passthru.tests jitsi-meet jitsi-meet-prosody jitsi-meet-prosody.passthru.tests jitsi-meet.passthru.tests jitsi-videobridge jitsi-videobridge.passthru.tests

ajs124 committed:
nixos/jicofo: fix after update

https://github.com/jitsi/jicofo/commit/2943c21ff7a9e2464a7be646fdf10a9bde4f8c6a
removed the cli parameters

migrate from legacy config while we're at it
